runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -1691,6 +1795,8 @@ J9::TransformUtil::transformIndirectLoadChainAt(TR::Compilation *comp, TR::Node
       }
 #endif /* defined(J9VM_OPT_JITSERVER) */
 
+   return false;
```

Comment:
Why is this early return here?

Reply:
`transformIndirectLoadChainAt` was broken when we changed the impl method to take the KnownObjectTable index rather than an address. I tried bodging a fix by simply adding any address that came into `transformIndirectLoadChainAt` into the KnownObjectTable, but that caused the garbage collector to freak out, so I had to bypass it for now.